Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
-ruby '2.3.1'
+ruby '2.1.5'
```

Comment:
any reason you bumped the ruby version down? it might be safer to make version changes separately.